<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketsWindow</class>
 <widget class="QMainWindow" name="TicketsWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AviasalesKiller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1011</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Flyght">
     <attribute name="title">
      <string>Главная</string>
     </attribute>
     <widget class="QTableView" name="Flying_Table">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>981</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="To_Edit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>331</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Куда</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="From_Edit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>331</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Откуда</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Search_Button">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>30</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="Date_Edit">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>30</y>
        <width>110</width>
        <height>26</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>7</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2024</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="time">
       <time>
        <hour>7</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::DaySection</enum>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="CheckData">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>30</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Все даты</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="profile">
     <attribute name="title">
      <string>Профиль</string>
     </attribute>
     <widget class="QLabel" name="user">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="mail">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableView" name="Buy_Ticket_Table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>981</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1036</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
